tests/cupyx_tests/scipy_tests/sparse_tests/test_construct.py

```diff
@@ -380,7 +380,7 @@ def test_diags_non_square(self, xp, sp):
 
 
 # borrowed from scipy:
-arrs = [[[0]],
+_arrs_kron = [[[0]],
```

Comment:
Maybe we need this to fix flake8?
```suggestion
_arrs_kron = [
        [[0]],
```